kit/popups/src/Sheets/UIBottomSheet.tsx

```diff
@@ -34,34 +48,44 @@ export function UIBottomSheet({
         );
     }, [style, bottomInset, hasDefaultInset]);
 
-    const sheetStyle = React.useMemo(() => {
-        return {
-            paddingBottom: defaultPadding + UILayoutConstant.rubberBandEffectDistance,
-        };
-    }, [defaultPadding]);
+    const styles = useStyles(defaultPadding, theme);
 
     return (
         <UISheet.Container visible={visible} forId={forId}>
             <UISheet.KeyboardAware defaultShift={-UILayoutConstant.rubberBandEffectDistance}>
                 <UISheet.IntrinsicSize>
-                    <UISheet.Content {...rest} style={[styles.bottom, style, sheetStyle]}>
-                        {children}
+                    <UISheet.Content {...rest} style={styles.sheet}>
+                        <View style={styles.card}>
```

Comment:
We need this so that user can style the container for the content (paddings, flex direction, alignment of elements)

Reply:
They can wrap their content with a View, and it would be granular, only when it's needed, in contrast now, it will be every time, even if user doesn't apply any customisations

Reply:
Oh, now I've got it, that's how you got rid of `left: 'auto'`

Reply:
Because `card` must be in the center and fill all available horizontal space till the `maxWidth`. But parent has default alignment styles.